cdap-hbase-compat-base/src/main/java/co/cask/cdap/data2/util/hbase/HBaseTableUtil.java

```diff
@@ -70,6 +70,8 @@
 
   public static final String CDAP_VERSION = "cdap.version";
 
+  public static final String HBASE_VERSION = "hbase.version";
```

Comment:
not a big deal, but maybe we should name it 'hbase.compat.version', in case some hbase admin mistakes this for something hbase specific.

Reply:
`hbase.compat.version` might be misleading, since we use `compat` term for the cdap modules compatible with the hbase versions on the cluster. One compat module can be compatible with the different versions of hbase. 

Here we want to actually use the hbase version from the cluster and not the compat module version from cdap. 

Should I rename it to `cdap.hbase.version` to make it specific that the property is owned by cdap?

Reply:
Oh I thought the compat version is what was being used. Shouldn't the compat version be used and not the exact hbase version? There's no need to upgrade the coprocessor if the cdap version is the same and the hbase compat version is the same.

Reply:
Better prefix it with cdap for any CDAP specific attribute. E.g cdap.hbase.version

Reply:
renamed to `cdap.hbase.version` and using the value of hbase compat module rather than the hbase version from cluster.